libs/server/Resp/KeyAdminCommands.cs

```diff
@@ -148,6 +148,36 @@ private bool NetworkEXPIRE<TGarnetApi>(RespCommand command, ref TGarnetApi stora
                 }
             }
 
+            if (parseState.Count > 3)
+            {
+                if (!parseState.TryGetEnum(3, true, out ExpireOption additionExpireOption) || !additionExpireOption.IsValid(ref parseState.GetArgSliceByRef(3)))
+                {
+                    var optionStr = parseState.GetString(3);
+
+                    while (!RespWriteUtils.WriteError($"ERR Unsupported option {optionStr}", ref dcurr, dend))
+                        SendAndReset();
+                    return true;
+                }
+
+                if (expireOption == ExpireOption.XX && (additionExpireOption == ExpireOption.GT || additionExpireOption == ExpireOption.LT))
```

Comment:
You're correct here that it is legal to specify both LT / GT and XX, and that an error message needs to be presented if there is an illegal combination of parameters, but there's no point in adding XXGT and XXLT, since they're essentially the same as GT, LT

Reply:
Answered in the above review comment. @TalZaccai  Let me know if you are fine with having XXGT and XXLT

Reply:
Yeah then it's ok. Can we drop XXGT though?

Reply:
Personally, I like to have it to differentiate it even in practice (as mentioned above) there is no difference as its a valid option. Your repo your rules, confirm me that I need to remove I will remove it :)